DirectProgramming/DPC++/DenseLinearAlgebra/matrix_mul/Makefile

```diff
@@ -1,8 +1,8 @@
 DPCPP_CXX = dpcpp
```

Comment:
@KanclerzPiotr Do we want to rename DPCPP to SYCL in the Makefile or maybe get rid of it altogether?